docs/sql.rst

```diff
@@ -219,80 +219,80 @@ field expressions will be automatically substituted. Here is long / short format
 This method is automatically used by :php:class:`FieldSqlExpression`.
 
 
-Actions
+Queries
 =======
 
-The most basic action you can use with SQL persistence is 'select'::
+The most basic query mode you can use with SQL persistence is 'select'::
 
-    $action = $model->action('select');
+    $query = $model->toQuery()->select();
```

Comment:
returning `$model->toQuery()` is very dangerous as it can not be reused for another query/mode

we should keep `$model->toQuery(MODE::...)`

Reply:
Included an exception if mode change attempted.
I prefer keeping the way to set the mode using methods.
If no mode specifically set then default is 'select'